### ISSUE TITLE
Total Resources count is wrong

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#5331]
I don't know what it is supposed to show, but there are clearly more than 68 resources:

![image](https://user-images.githubusercontent.com/1955897/157466436-4fa4eb4a-4cf3-4c73-a241-065f980d3b40.png)

**Comment by nwmac** [comment]
This was with a 2.6.4 backend - but I don't think anything has changed.

**Comment by Shavindra** [comment]
@nwmac The resource count is showing this list 
```
[
    "namespace",
    "networking.k8s.io.ingress",
    "persistentvolume",
    "apps.deployment",
    "apps.statefulset",
    "batch.job",
    "apps.daemonset",
    "service"
]
```

**Comment by markusewalker** [comment]
Verified this is addressed in `v2.6-head` https://github.com/rancher/rancher/commit/7a88105e02d8b779a486a544cc36b599b498d458 .

**TEST RESULT**
PASS

**REPRODUCTION STEPS**
1. Setup Rancher `v2.6.3` and navigated to Rancher > top-left dropdown menu > Explore Cluster > local.
2. Verified the Cluster Dashboard looks incorrect:
![image](https://user-images.githubusercontent.com/52591819/164744861-bd6f23da-f9bf-4896-8237-b619e12c8d77.png)

**VERIFICATION STEPS**
1. Setup Rancher `v2.6.3` and navigated to Rancher > top-left dropdown menu > Explore Cluster > local.
2. Verified the Cluster Dashboard looks correct now:
![image](https://user-images.githubusercontent.com/52591819/164744950-d21fc834-d756-4b11-a74a-2178c455acd5.png)
